src/Symfony/Component/Security/Core/Authorization/ExpressionLanguageProvider.php

```diff
@@ -14,6 +14,10 @@
 use Symfony\Component\ExpressionLanguage\ExpressionFunction;
 use Symfony\Component\ExpressionLanguage\ExpressionFunctionProviderInterface;
 
+if (!interface_exists('Symfony\Component\ExpressionLanguage\ExpressionFunctionProviderInterface')) {
+    throw new \RuntimeException('The ExpressionLanguage class requires the "ExpressionLanguage" component. Install "symfony/expression-language" to use it.');
```

Comment:
The name of this class is "ExpressionLanguageProvider".

Reply:
Sorry, I've done in a hurry, like a monkey with copy and paste.